packages/datadog-plugin-redis/test/legacy.spec.js

```diff
@@ -99,36 +99,41 @@ describe('Plugin', () => {
           client.stream.destroy()
         })
 
-        it('should handle errors', done => {
-          const assertError = () => {
-            if (!error || !span) return
-
-            try {
-              expect(span.meta).to.have.property(ERROR_TYPE, error.name)
-              expect(span.meta).to.have.property(ERROR_MESSAGE, error.message)
-              expect(span.meta).to.have.property(ERROR_STACK, error.stack)
-              expect(span.meta).to.have.property('component', 'redis')
-              expect(span.metrics).to.have.property('network.destination.port', 6379)
-              done()
-            } catch (e) {
-              done(e)
+        describe('known flakey test', () => {
+          this.retry(5)
```

Comment:
I think it needs to be a `function` for `this.retries` to work (not `this.retry`)
```suggestion
        describe('known flakey test', function () {
          this.retries(5)
```
also I think this can be set at the test itself, it doesn't need to be wrapped in a `describe`